blockexplorer/store/migrations/0004_txtype_index.sql

```diff
@@ -0,0 +1,81 @@
+
+-- +goose Up
+ALTER TABLE tx_results ADD column tx_type TEXT;
+ALTER TABLE tx_results ADD column sender TEXT;
+ALTER TABLE tx_results ADD column receiver TEXT;
+CREATE INDEX tx_results_tx_type_sender_index ON tx_results(tx_type, sender);
```

Comment:
@EVODelavega Hey, I'm sorry, seems your comment somehow fell through the cracks... 
Look, those are all very valid points. However, taking into account that the Tendermint indexer will apparently be removed from Tendermint and left to be implemented at the app level, this PR brings but a workaround, until we get to implement it in the app level, which might happen sooner rather than later. I think it would be acceptable to make sure that this change will not get in the way and slow down the validator performance further, and we kind of did that by running performance tests (thnx @peterbarrow) and comparing the results to those of the current develop branch. So, I wouldn't look too much into it right now.